src/stubs.cpp

```diff
@@ -438,4 +455,64 @@ extern "C" {
     StringPiece str(String_val(v_str), caml_string_length(v_str));
     CAMLreturn(caml_copy_string(RE2::QuoteMeta(str).c_str()));
   }
+
+
+  CAMLprim value mlre2__create_set(value v_options) {
+    value v_retval;
```

Comment:
Please use the "simple interface" described in the manual 19.5.1: https://caml.inria.fr/pub/docs/manual-ocaml/intfc.html#sec423